recipes/easy_profiler/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class EasyProfilerConan(ConanFile):
+    name = "easy_profiler"
+    description = "Lightweight profiler library for c++"
+    license = "MIT"
+    topics = ("conan", "easy_profiler")
+    homepage = "https://github.com/yse/easy_profiler/"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True
+    }
+    short_paths = True
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        # The windows build seems to be giving problems due to certain symbols
+        # not being exported properly for static libraries.
+        if not self.options.shared and self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Must be built as shared on \
+              Windows")
```

Comment:
On Windows, if static, does it fail at build or consume time?

As consume time, seems that `EASY_PROFILER_STATIC` must be defined if static and Windows: https://github.com/yse/easy_profiler/blob/046d9bd40cfd55ca3b1d798b16ad2a5673f66806/easy_profiler_core/include/easy/details/easy_compiler_support.h#L56

Reply:
I tried that, but it seems there are still some unresolved symbols.

Reply:
Ok I tried your branch. Static seems fine, but `psapi` must be added to `cpp_info.system_libs` for windows (unresolved symbols to `GetModuleBaseNameA`).

Reply:
```suggestion
```

Reply:
@SpaceIm , could you test it again using ( https://github.com/yse/easy_profiler/blob/develop/sample/main.cpp ) as the test package code. I was using this for the conan test_package and that's when it was failing for me. I then switched to a simpler example as per @uilianries  request.

Reply:
No problem on my computer, it works on Windows 10 if static.

Reply:
Perfect, add your changes then!